compiler/src/iree/compiler/Codegen/SPIRV/KernelConfig.cpp

```diff
@@ -575,14 +623,21 @@ LogicalResult setMatmulOpConfig(spirv::ResourceLimitsAttr limits,
   // keep the default for compilation configs that don't specify a pipeline
   // depth.
   auto pipelineDepth = softwarePipelineDepth ? softwarePipelineDepth : 1;
+  auto storeStage = softwarePipelineStoreStage;
+
+  // No multi-buffering if fused with a non-matmul op.
+  if (isFusedWithUnbufferableOp(op)) {
```

Comment:
Can we add a test for this case?

Reply:
Ah this comment is from before I figured out why it was failing for a specific case. I can add the above mlir as a test.

Reply:
+1

Reply:
This ended up being unnecessary after removing distribution loops before multi-buffering